es/tabs/work.md

```diff
@@ -0,0 +1,178 @@
+            Estudios Eslavos de la UCL.
+        </p>
+        </div>
+        <div class="containerMember">
+            <img  class="imgMembers" src="/assets/img/home/Rubi.webp" alt="Photo of Jule Rubi"/>
+            <h3>Jule Rubi</h3>
+            <h4 class="job">Jefa de proyectos culturales y Curadora</h4>
+                <p class="desc">Como curador, lector ferviente y viajero, la pasión de Jule es contar
+                historias del mundo tal como es hoy a través de objetos y obras de
+                arte. Tras su formación en la Ecole du Louvre y el Musée des Arts
+                et traductions Populaires (hoy MuCEM de Marsella), viajó a
+                Montreal como estudiante, antes de continuar su carrera en
+                Londres y Mascate, Omán. Anteriormente curadora y productora de
+                exposiciones en el Victoria and Albert Museum de Londres, pasó a
+                contar historias musicales en la Royal Opera House de Omán y
+                luego se unió a la agencia de diseño británica Cassonmann para la
+                concepción del Musée national de la Marine en París. Fascinada
+                por los idiomas y el puente entre culturas que es la traducción,
+                también trabajó con la organización sin fines de lucro ATLAS y el
+                Institut Français para promover la traducción de la literatura árabe
+                en Europa, donde descubrió el nutritivo mundo de las residencias
+                artísticas. Sus diversas habilidades enriquecen las estructuras
+                culturales internacionales que trabajan en la intersección entre las
+                artes, la música y la literatura.</p>
+        </div>
+        <div class="containerMember">
+            <img class="imgMembers" src="/assets/img/home/caroRes.webp" alt="Photo of Caroline Trotman"/>
+            <h3>Caroline Trotman</h3>
+            <h4 class="job">Historiador y curadora</h4>
+                <p class="desc">Caroline es una profesora motivadora y analítica de Historia,
+                Geografía, Geopolítica y Ciencias Políticas en París, Francia. En
+                2009-15, vivió entre Florida y California, EE. UU., luego Londres y
+                Oxford, Reino Unido, donde trabajó como curadora adjunta y
+                registradora en varios museos, y así conoció a Elisa. Juntos
+                colaboraron en un proyecto de investigación sobre el monumento de
+                Buzludzha en Bulgaria entre 2014 y 2016, del que surgieron un libro,
+                artículos y una exposición. Caroline tiene una licenciatura en
+                Historia, Historia del Arte y Geografía de la Sorbona, París, y tres
+                maestrías: en Arte Moderno e Inglés de la Universidad Oxford
+                Brookes, en Estudios de Museos de la Universidad de Leicester y en
+                Teoría del Arte Contemporáneo de Goldsmiths, Universidad de
+                Londres. (todo el Reino Unido).
```

Comment:
Remove the stop in `Universidad de Londres. (todo el Reino Unido).`. It should look like this:
``Universidad de Londres (todo el Reino Unido).``